hypervisor/include/arch/x86/vtd.h

```diff
@@ -588,8 +588,8 @@ void resume_iommu(void);
  *
  * Register DMAR units on the platform according to the pre-parsed information or DMAR table.
  *
- * @return 0 - on success
- * @return non-zero - iommu is a must have feature, if init_iommu failed, the system should not continue booting
+ * @retval 0 on success
```

Comment:
```suggestion
 * @retval 0 on success, non-zero otherwise. The IOMMU is a must-have feature, if init_iommu failed, the system should not continue booting
```

Reply:
Having the success and failure cases separated as two `@retval` seems more explicit to me. How about updating the `non-zero` with `<0`, like we are doing above?